src/defs/clientPolicyRepresentation.ts

```diff
@@ -0,0 +1,13 @@
+import ClientPolicyConditionRepresentation from './clientPolicyConditionRepresentation';
+
+/**
+ * https://www.keycloak.org/docs-api/15.0/rest-api/#_clientpolicyrepresentation
+ */
+
```

Comment:
Can you remove this newline so the comment is aligned with the interface? That way it will be picked up as JSDoc.